### DISCUSSION TITLE
Hello Everyone!

### DISCUSSION THREAD

**Discussion by AmanPriyanshu** [OpenMined/courses#31] [Introduce Yourself]
Hi, I'm Aman Priyanshu, an undergrad student with a keen interest for Machine Learning and Cyber Security, two fields that really click well with Federated Learning and Privacy Enabled Learning. Hope to learn a lot from this course. A great start for the New Year! really excited to learn and grow in this community.

**Comment by yemikifouly** [comment]
Hello Aman! 👋 With the Private AI Series, you will learn so much more than Federated Learning. Given your areas of interest I am sure that you will enjoy the first course and the upcoming courses. 🙂 Welcome to the discussion board and remember to post questions you have when going through the course.
